src/Sarif/Visitors/SarifVersionOneToCurrentVisitor.cs

```diff
@@ -9,218 +9,177 @@ namespace Microsoft.CodeAnalysis.Sarif.Visitors
-            "text/tsv",
-            "text/uri-list",
-            "text/x-asm",
-            "text/x-c",
-            "text/x-csharp",
-            "text/x-h",
-            "text/x-java-source",
-            "text/x-java-source,java",
-            "text/xml",
-            "text/x-pascal"
-        };
-        #endregion
-
-        private static readonly Dictionary<AlgorithmKindVersionOne, string> s_AlgorithmKindNameMap = new Dictionary<AlgorithmKindVersionOne, string>
-        {
-            { AlgorithmKindVersionOne.Sha1, "sha-1" },
-            { AlgorithmKindVersionOne.Sha3, "sha-3" },
-            { AlgorithmKindVersionOne.Sha224, "sha-224" },
-            { AlgorithmKindVersionOne.Sha256, "sha-256" },
-            { AlgorithmKindVersionOne.Sha384, "sha-384" },
-            { AlgorithmKindVersionOne.Sha512, "sha-512" }
-        };
-
         public SarifLog SarifLog { get; private set; }
 
-        public override SarifLogVersionOne VisitSarifLogVersionOne(SarifLogVersionOne node)
+        public override SarifLogVersionOne VisitSarifLogVersionOne(SarifLogVersionOne v1SarifLog)
         {
             SarifLog = new SarifLog(SarifVersion.TwoZeroZero.ConvertToSchemaUri(),
                                     SarifVersion.TwoZeroZero,
                                     new List<Run>());
 
-            foreach (RunVersionOne run in node.Runs)
+            foreach (RunVersionOne run in v1SarifLog.Runs)
             {
                 VisitRunVersionOne(run);
             }
 
             return null;
         }
 
-        public static FileData CreateFileData(FileDataVersionOne node)
+        public static FileData CreateFileData(FileDataVersionOne v1FileData)
         {
             FileData fileData = null;
 
-            if (node != null)
+            if (v1FileData != null)
             {
                 fileData = new FileData
                 {
-                    Length = node.Length,
-                    MimeType = node.MimeType,
-                    Offset = node.Offset,
-                    ParentKey = node.ParentKey,
-                    Properties = node.Properties                    
+                    Length = v1FileData.Length,
+                    MimeType = v1FileData.MimeType,
+                    Offset = v1FileData.Offset,
+                    ParentKey = v1FileData.ParentKey,
+                    Properties = v1FileData.Properties
                 };
 
-                if (node.Uri != null)
+                if (v1FileData.Uri != null)
                 {
                     fileData.FileLocation = new FileLocation
                     {
-                        Uri = node.Uri,
-                        UriBaseId = node.UriBaseId
+                        Uri = v1FileData.Uri,
+                        UriBaseId = v1FileData.UriBaseId
                     };
                 }
 
-                fileData.Contents = new FileContent
+                if (!string.IsNullOrWhiteSpace(v1FileData.Contents))
```

Comment:
Didn't we settle on just checking for `null` here, because the empty string `""` is valid? And I think I said something like "Don't bother checking for an all-white-space string. There are lots of strings that aren't valid MIME Base64, like ")(*^%&*(", and we're not checking for those".

Reply:
Yup, fixed